vaadin-checkbox-flow-integration-tests/src/test/java/com/vaadin/flow/component/checkbox/tests/DataViewIT.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.checkbox.tests;
+
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+@TestPath("vaadin-checkbox-group-test-data-view")
+public class DataViewIT extends AbstractComponentIT {
```

Comment:
Do we need DataViewIT and ListDataViewIT when they both use ListDataView?

Reply:
Initially the idea was to separate dataview and listdataview API in tests. But, you're right, they both use CheckboxGroupListDataView and I believe it should be combined into one test class.